Src/BootCamp.Chapter/ArrayOps.cs

```diff
@@ -0,0 +1,107 @@
+﻿namespace BootCamp.Chapter
+{
+    public static class ArrayOps
+    {
+        public static int FindMaxValue(int[] inputArray)
+        {
+            if (IsNullOrEmpty(inputArray))
+            {
+                return default;
+            }
+            int max = inputArray[0];
+            for (int i = 0; i < inputArray.Length; i++)
```

Comment:
It makes no difference but this could have been starting from 1. x)

Reply:
Indeed.  I changed it to 0 when I encountered a bug and forgot to change it back. Thank you!
I never mind constructive feedback and if you have time, patience I would read and learn from your feedback... I would not want to move forward with basics learnt wrong.
Thank you for everything you do for us! Cheers!

Reply:
I fixed it [7600168](https://github.com/csinn/CSharp-From-Zero-To-Hero/pull/175/commits/76001687331e06de12bb26977c18a012a52e5c71)